charts/bindplane/templates/_helpers.tpl

```diff
@@ -37,9 +37,6 @@ The image tags to use
 {{- define "bindplane.tag" -}}
 {{- printf "%s" (default (printf "%s" .Chart.AppVersion) .Values.image.tag) }}
 {{- end -}}
-{{- define "prometheus.tag" -}}
-{{- printf "%s" (default (printf "%s" .Chart.AppVersion) .Values.prometheus.image.tag) }}
-{{- end -}}
```

Comment:
We used this helper to set the Prometheus image based on the Chartfile's application version (BindPlane's version). This is no longer nessisary because the values file has a default value for `prometheus.image.tag`.